src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -570,6 +626,23 @@ class MFEMSidreDataCollection : public mfem::DataCollection
   void createMeshBlueprintAdjacencies(bool hasBP);
   #endif
 
+  /// Retrieves a pointer to the View onto the data for a Field
+  /// For vector-valued Fields, retrieves the view to the first component
+  /// as data is interleaved
+  View* getFieldValuesView(const std::string& field_name);
```

Comment:
I haven't looked at the implementation yet to see how this is used, but should this function take the index of the component that you want to get? Perhaps with a default of getting the first component?

Reply:
This is just used to obtain the raw `double*` pointer to the data, so for a vector-valued field the view to the first component is always sufficient